watch.go

```diff
@@ -219,6 +224,8 @@ func Start(appname string) {
 		appname = "./" + appname
 	}
 
+	formatSourceCode(routersPackage)
```

Comment:
Urgh I see bee provides a function to format the Go code via exec call 🤯 Well ok 🤷‍♂️